docs/macios/xcsync.md

```diff
@@ -20,68 +26,102 @@ Supported file types include:
 
 The tool has two commands: `generate` and `sync`. Use `generate` to create an Xcode project from a .NET project and `sync` to bring changes in the Xcode project back to the .NET project.
 
-## Synopsis
+<!-- markdownlint-disable MD025 -->
+# [Visual Studio Code](#tab/visual-studio-code)
```

Comment:
Somewhere in this section we need to indicate that the commands are only available when using VS Code on macOS.

Reply:
thought this was clear since Xcode is necessary, but added a disclaimer